server/handler/base.go

```diff
@@ -202,20 +202,25 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, pe
 	}
 
 	if performActions && statusState == "pending" && !prctx.IsDraft() {
-		hasReviewers, err := prctx.HasReveiwers()
+		hasReviewers, err := prctx.HasReviewers()
 		if err != nil {
 			logger.Warn().Err(err).Msg("Unable to list request reviewers")
 		}
 
-		if !hasReviewers {
+		hasReviews, err := prctx.HasReviews()
+		if err != nil {
+			logger.Warn().Err(err).Msg("Unable to list reviews")
```

Comment:
Maybe this should return an error so that we return and stop processing?

Reply:
That seems better to me, since the end result is the same... logging an error doesn't allow any best effort attempt later on.